dataform/examples/dataform_datavault/README.md

```diff
@@ -0,0 +1,107 @@
+
+### Source table schemas:
+Meter Staging table:
+
+  CREATE OR REPLACE TABLE staging.meter_load
+  (
+  meter_id INTEGER,
+  manufacturer STRING,
+  service_name STRING,
+  source STRING,
+  loaddate DATE
+  );
+
+Meter Readings Staging table:
+
+  CREATE OR REPLACE TABLE staging.meter_readings_load
+  (
+  readings_id INTEGER,
+  meter_id INTEGER,
+  readings FLOAT64,
+  readings_time TIMESTAMP,
+  source STRING,
+  loaddate DATE
+  );
+
+### Source table exports:
+1. meter_load --> Meter table contains data about meter.
+   ![](screenshots/meter_load.png)
+
+2. meter_readings_load --> Meter readings table contains data about meter readings.
+   ![](screenshots/meter_readings_load.png)
+
+## How to use?
+
+### Pre-requsites:
+1. npm lib needs be installed
+2. [Google cloud project](https://developers.google.com/workspace/guides/create-project) should be available
+3. BigQuery api is enabled
+4. Access to create service account keys
+5. [Dataform lib ](https://docs.dataform.co/dataform-cli) lib using NPM
+
+
+### Steps to follow:
+All the steps below can be used via terminal or Use the CloudShell editor.
+
+In order to test and run the dataform, please follow below steps:
+
+1. Initialize a Dataform project and then create a new Dataform google cloud project ([*optional if you already have one*])
+
+- dataform init bigquery dataform-demo --default-database $(gcloud config
+  get-value project) --include-schedules
+
+2. Set up authentication to BigQuery
+    - Go to https://console.cloud.google.com/iam-admin/serviceaccounts and
+      create a service account. Give this account the role of BigQuery Admin (it helps Dataform can create new tables etc.). Then, download the JSON key to
+      the project and upload the file to CloudShell. (optional if you already have a account with service account key uploaded into *optional if you already have a account with service account key uploaded into it*)
+    - Using terminal:
+        - cd dataform-demo
+        - dataform init-creds bigquery
+    - Provide the path to the JSON key file when asked. Then make sure to add
+      the key file to .gitignore so that you don’t check it in by mistake.
+      - echo filename.json > .gitignore
+      - git add -f .gitignore
+3. Set up raw sources for meter & meter_readings. Definitions & functions to create DataVault tables in BigQuery. 
+    - Let’s start out by adding a definition for a source named
+      definitions/sources/meter.sqlx and 
+      definitions/sources/meter_readings.sqlx
+    - Function used in sql query can be placed under includes/hashing.js
+    - DataVault structure 
+    
+      <img src="screenshots/datavault_tables.png" width=250>
+    
+4. Compile and Dry Run
+    - dataform compile
+5. Run
+    - dataform run
```

Comment:
code block

Reply:
Here you may add the GIF so that everyone knows what to expect as the output. Make sure that in GIF there is nothing related to customers.